R/getNumSum.R

```diff
@@ -0,0 +1,36 @@
+getNumSum = function(data, features, target) {
```

Comment:
Even if this is a internal function, it would be nice to make a description in roxygenstyle. so you can avoid the ::operator

Reply:
ok. will do it